gcc.symbols.imp

```diff
@@ -148,7 +148,15 @@
   { symbol: [ "suseconds_t", private, "<sys/select.h>", public ] },
   { symbol: [ "suseconds_t", private, "<sys/time.h>", public ] },
   { symbol: [ "time_t", private, "<sys/types.h>", public ] },
+  { symbol: [ "time_t", private, "<sched.h>", public ] },
+  { symbol: [ "time_t", private, "<sys/msg.h>", public ] },
+  { symbol: [ "time_t", private, "<sys/select.h>", public ] },
+  { symbol: [ "time_t", private, "<sys/sem.h>", public ] },
+  { symbol: [ "time_t", private, "<sys/shm.h>", public ] },
+  { symbol: [ "time_t", private, "<sys/stat.h>", public ] },
+  { symbol: [ "time_t", private, "<sys/time.h>", public ] },
   { symbol: [ "time_t", private, "<time.h>", public ] },
```

Comment:
It seems to me we'd get slightly better portability by putting `<time.h>` first for `time_t`.

Windows systems don't have a `sys/types.h`, but they do have `time.h`, for example.

Reply:
Agreed.